### DISCUSSION TITLE
de/2-11-kata-10/

### DISCUSSION THREAD

**Discussion by giscus** [cogneon/lernos-ai#62] [General]
# de/2-11-kata-10/

https://ai.lernos.org/de/2-11-kata-10/

**Comment by ghost** [comment]
Im Satz "Lese Dir zunächst das Kapitel Prompts erstellen!" (2. Absatz) entweder das Wort "Dir" löschen oder das Wort "durch" am Ende ergänzen.

**Comment by ghost** [comment]
Die Nutzung von mehrstufigen Prompts eignet sich um komplexe Aufgabenstellungen und Anfragen zu bearbeiten. → Komma hinter dem Wort "sich" ergänzen.

Beginnen mit einer klaren Definition des Ziels. → Beginne mit einer klaren Definition des Ziels.

Überlege Dir 3 praktische Übungen, ... → Überlege Dir drei praktische Übungen, ...
